tests/testthat/test-groupby.R

```diff
@@ -7,6 +9,19 @@ df = pl$DataFrame(
 
 gb = df$group_by("foo", maintain_order = TRUE)
 
+test_that("groupby", {
+  df2 = gb$agg(
+    pl$col("bar")$sum()$alias("bar_sum"),
+    pl$col("bar")$mean()$alias("bar_tail_sum")
+  )$to_data_frame()
+
+  expect_equal(
+    df2,
+    data.frame(foo = c("one", "two"), bar_sum = c(9, 6), bar_tail_sum = c(4.5, 2))
+  )
+})
+
+
```

Comment:
I only moved this above because otherwise RStudio doesn't provide the button "Test"